locales/locales.php

```diff
@@ -1790,6 +1815,14 @@ public function __construct() {
 		$sq->slug = 'sq';
 		$sq->google_code = 'sq';
 		$sq->facebook_locale = 'sq_AL';
+		
+		$sq_XK = new GP_Locale();
+		$sq_XK->english_name = 'Shqip (Kosovo)';
+		$sq_XK->native_name = 'Për Kosovën Shqip';
+		$sq_XK->lang_code_iso_639_3 = '';
```

Comment:
Sorry, being a bit dumb here but does that mean I should remove it? If yes, I guess that also answers my question about Spanish above.